### DISCUSSION TITLE
Fantastical

### DISCUSSION THREAD

**Discussion by Lmasey** [open-horizon-labs/obsidian-ics#110] [Q&A] [answered]
Does this work with Fantastical? If so which link do I use?

**Comment by muness** [comment]
Sorry, I don't follow. I think Fantastical is a calendar client, not a server. Do you use it to create new calendars or do you connect to existing icloud, google, o365 calendars?

**Reply by Lmasey** [reply]
I add my calendars to it. I know it has an x-url and was hoping it would work:)

**Reply by muness** [reply]
As in pull the calendar events from your local Fantastical client?

**Reply by Lmasey** [reply]
Yes:) Right now I'm manually copying and pasting.

**Comment by muness** [comment, marked as answer]
I don't see any way to access the Fantastical data store. 

If they store the data as ical files / folders, you can use the `vdir` feature to get access without any other support in the plugin but I doubt it.

My suggestion is to set up the same calendars using the plugin to avoid relying on Fantastical specific implementation that is prone to change over time.
